src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(a) {
+    return b => {
+      count++;
+
+      return count <= 3 || count % 2 !== 0 ? a + b : 'Bzzz... Error!';
```

Comment:
In the current implementation, the condition `count % 2 !== 0` will return the sum for odd calls after the third call. It should return the correct sum for odd calls after the third call, not even.